pex/bin/pex.py

```diff
@@ -872,6 +868,9 @@ def seed_cache(
 
     pex_path = cast(str, options.pex_name)
     with TRACER.timed("Seeding local caches for {}".format(pex_path)):
+        # NB: This pattern causes the defaulted PEX_ROOT property to overwrite any runtime_pex_root in the written PEX-INFO.
```

Comment:
Seeding is necessarily a local operation / optimization. As far as I know, only Pants uses this to pre-seed PEX venvs as the PEXes are built for local venvs it uses to execute Python tools. The whole mechanism was exposed before `--sh-boot` and `--sh-boot` can proably replace it now.

With that background - most importantly the local bit - can your NB turn into a bug with an issue or should it go away entirely? I don't like the idea of an NB staying in the codebase that, by language, appears to point out a maybe bug. I'd like to complete the analysis and file a bug or realize it's not if possible.

Reply:
Makes sense. I didn't understand the ramifications of changing seed pex / didn't want to fix it in this change. I can write up a bug and change to a TODO.

Reply:
Ok, if you're sure its actually a bug for the local case. Seeding is only useful if the local pex root is writeable.

Reply:
In that case perhaps not a bug.